src/test/java/me/itzg/helpers/modrinth/InstallModrinthModpackCommandTest.java

```diff
@@ -0,0 +1,178 @@
+package me.itzg.helpers.modrinth;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static me.itzg.helpers.modrinth.ModrinthTestHelpers.*;
+import static com.github.tomakehurst.wiremock.client.WireMock.*;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.nio.file.*;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+
+import com.github.tomakehurst.wiremock.junit5.WireMockRuntimeInfo;
+import com.github.tomakehurst.wiremock.junit5.WireMockTest;
+
+import me.itzg.helpers.files.Manifests;
+import me.itzg.helpers.modrinth.model.*;
+import me.itzg.helpers.modrinth.model.ModpackIndex.ModpackFile;
+
+@WireMockTest
+public class InstallModrinthModpackCommandTest {
+    private String projectName = "test_project1";
+    private String projectId = "efgh5678";
+    private String projectVersionId = "abcd1234";
+    private Version projectVersion =
+        createModrinthProjectVersion(projectVersionId);
+
+    static InstallModrinthModpackCommand createInstallModrinthModpackCommand(
+        String baseUrl, Path outputDir, String projectName, String versionId,
+        ModpackLoader loader)
+    {
+        InstallModrinthModpackCommand commandUT =
+            new InstallModrinthModpackCommand();
+        commandUT.baseUrl = baseUrl;
+        commandUT.outputDirectory = outputDir;
+        commandUT.modpackProject = projectName;
+        commandUT.version = versionId;
+        commandUT.loader = loader;
+
+        return commandUT;
+    }
+
+    @Test
+    void downloadsAndInstallsModrinthModpack(
+            WireMockRuntimeInfo wm, @TempDir Path tempDir
+        ) throws JsonProcessingException, IOException, URISyntaxException
+    {
+        String expectedFileData = "some test data";
+        String relativeFilePath = "test_file";
+        ModpackFile testFile = createHostedModpackFile(
+            relativeFilePath, expectedFileData, wm.getHttpBaseUrl());
+
+        ModpackIndex index = createBasicModpackIndex();
+        index.getFiles().add(testFile);
+
+        stubModrinthModpackApi(
+            wm, projectName, projectId, projectVersion,
+            createModrinthPack(index));
+
+        InstallModrinthModpackCommand commandUT =
+            createInstallModrinthModpackCommand(wm.getHttpBaseUrl(), tempDir,
+                projectName, projectVersionId, ModpackLoader.forge);
+
+        int commandStatus = commandUT.call();
+
+        assertEquals(0, commandStatus);
+
+        String actualFileData =
+            new String(Files.readAllBytes(tempDir.resolve(relativeFilePath)));
+
+        assertEquals(expectedFileData, actualFileData);
+    }
```

Comment:
You still need to convert to AssertJ. They have some very nice `Path` (file content, etc) assertions:

https://www.javadoc.io/doc/org.assertj/assertj-core/latest/org/assertj/core/api/AbstractPathAssert.html